src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -51,8 +57,8 @@ protected EnvVars buildEnv(AbstractProject<?, ?> job) {
             for (EnvironmentContributor contributor : EnvironmentContributor.all()) {
                 try {
                     contributor.buildEnvironmentFor(job, env, TaskListener.NULL);
-                } catch (Exception e) {
-                    // ignore
+                } catch (Throwable e) {
+                    LOGGER.warn("Unhandled exception. Skipping environment contributor " + contributor, e);
```

Comment:
this is slf4j so used full parameterized logging as it doesn;t have the same limitaion as JUL with parameterized and throwables.

However I think you actually mean this to be JUL.